api/src/pcapi/core/educational/api/dms.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from datetime import timedelta
+import logging
+import typing
+
+from pcapi import settings
+from pcapi.connectors.dms import api as dms_api
+from pcapi.connectors.dms import models as dms_models
+from pcapi.core.educational import models as educational_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def update_dms_status(procedure_id: int = 0) -> None:
+    """import dms applications for eac status"""
+    if procedure_id:
+        procedures = [procedure_id]
+    else:
+        procedures = [
+            settings.DMS_EAC_PROCEDURE_INDEPENDANTS_CANDIDATE_ID,
+            settings.DMS_EAC_PROCEDURE_STRUCTURE_CANDIDATE_ID,
+            settings.DMS_EAC_PROCEDURE_MENJS_CANDIDATE_ID,
+        ]
+
+    for procedure in procedures:
+        previous_import = _get_previous_import(procedure)
+        if previous_import and previous_import.isProcessing:
+            logger.info(
+                "[DMS] Procedure %s is already being processed.",
+                procedure,
+                extra={
+                    "procedure_number": procedure,
+                },
+            )
+            continue
+        since = previous_import.latestImportDatetime if previous_import else None
+
+        _update_dms_status_for_procedure(
+            procedure_number=procedure,
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏`procedure_id` ou `procedure_number` il faut pas mélanger les 2 !
Dans DMS les 2 existent et ne veulent pas dire la même chose. Nous on utilise le `procedure_number`